packages/MarkdownEditor-Core.package/MarkdownInlineParser.class/instance/parse..st

```diff
@@ -3,9 +3,9 @@ parse: aStringOrText
 
 	^ self 
 		clean;
-		text: aStringOrText asText;
+		textToParse: aStringOrText asText;
 		tokenize;
 		interpretTokens;
 		attributizeText;
-		text
+		textToParse
```

Comment:
Here it becomes very weird that the attribute is called `textToParse` because while in line 6 it is still a text that shall be parsed in line 10 the text already has been parsed, which is hidden by the name of the attribute. Sadly we had to rename it because the new super class already has a `text` but if you have a better idea for the name that would be great!

Reply:
Well but `textToStyle` doesn't make sense either because in line 10 it is already styled. Thus at that point it is no longer a text to be styled.